WordPress/Resources/it.lproj/Localizable.strings

```diff
@@ -3124,6 +3133,9 @@ translators: Block name. %s: The localized block name */
 /* Label for the file type (.JPG, .PNG, etc) for a media asset (image / video) */
 "File type" = "Tipo di file";
 
+/* Film & Television site intent topic */
+"Film & Television" = "Film e televisione";
```

Comment:
This might be a nitpick but I wonder if these translations should have kept the formatting from the English source?

I mean:

```suggestion
"Film & Television" = "Film & Televisione";
```

There's difference in the visual impact between the two options. The translations deviate from the original formatting all in the same way, so the end result will be homogeneous, but I wonder if it won't look as good as the English version.